Diary/Controller/DiaryViewController.swift

```diff
@@ -81,10 +88,13 @@ final class DiaryViewController: UIViewController {
     
     private func fetchDiaryContents() {
         do {
-            try diaryManager.fetchDiaryContents(name: "sample")
+            try diaryManager.fetchDiaryContents()
         } catch {
-            print(error.localizedDescription)
-            presentAlertWith(title: "데이터 불러오기 실패", message: "앱을 다시 실행해주십시오.", actionConfigs: ("확인", .default, nil))
+            os_log("%@", error.localizedDescription)
+            presentAlertWith(title: String(localized: "failedFatchDataAlertTitle"),
```

Comment:
이것도 갠취긴 한데,
            with는 아규먼트 레이블로 올 때 가독성이 더 좋은거 같아요.
            또한 localizing하는 부분도 "stringToLocalize".localized
            와 같은 extension을 만들어 쓰는걸 고려해볼 수 있을거 같아요
            ```suggestion
            let alertConfiguration = ...
            self.presentAlert(with: configuration)
            ```

Reply:
String extension에 localized 프로퍼티를 추가해서 수정해보았습니다.
덕분에 훨씬 깔끔해졌습니다 감사합니다! 🙇‍♀️